pkg/db-repo/notification.go

```diff
@@ -134,3 +134,26 @@ func getDefaultNotifierMetrics(modelId mdl.ModelId) []*metric.Datum {
 		},
 	}
 }
+
+type customAttributesNotifier struct {
+	baseNotifier
+	attributes map[string]interface{}
+}
+
+func (c *customAttributesNotifier) Send(ctx context.Context, notificationType string, value ModelBased) {
+	valueWithAttributes := attributesModelBased{
+		attributes: c.attributes,
+		ModelBased: value,
+	}
+
+	c.baseNotifier.Send(ctx, notificationType, valueWithAttributes)
```

Comment:
Will sending something of a private type not cause the line `out := n.transformer("api", n.version, value)` in the base notifier to fail? Because that passes the value to the transformer, which normally starts by asserting a type